content/references/api-reference/component-apis/createContext.mdx

```diff
@@ -39,4 +39,43 @@ export function CounterProvider(props) {
 }
 ```
 
-The value passed to provider is passed to `useContext` as is. That means wrapping as a reactive expression will not work. You should pass in Signals and Stores directly instead of accessing them in the JSX.
\ No newline at end of file
+The value passed to provider is passed to `useContext` as is. That means wrapping as a reactive expression will not work. You should pass in Signals and Stores directly instead of accessing them in the JSX.
+
+## Default value
+
+`createContext()` takes an optional "default value" as an argument. If `useContext` is called and there is no corresponding
+context provider above it in the component hierarchy, then the value passed as `defaultValue` is returned.
+However, if no `defaultValue` was passed, then `undefined` is returned in this case. Also, `defaultValue` (or `undefined`) is
+returned if `useContext` is called inside an event callback, as it is then outside of the component hierarchy.
+
+This has implications for TS. If no `defaultValue` is passed, then it is possible that `useContext()` will return `undefined`,
+and the types reflect this.
+
+## Common issues with `createContext` and `useContext`
```

Comment:
I think it would be better to move this part to the useContext page. It has to do more with useContext than createContext

Reply:
I can move it to there, but I felt it was a natural flow from the previous section discussing the default value, which is the basis of this section, and I reference to this page from the `useContext` page instead of splitting the material across two pages. What do you think?